src/layers/XRLayer/XRLayer.js

```diff
@@ -146,12 +146,23 @@ export default class XRLayer extends Layer {
   draw({ uniforms }) {
     const { textures, model } = this.state;
     if (textures && model) {
-      const { sliderValues, colorValues, opacity } = this.props;
+      const { sliderValues, colorValues, opacity, dtype } = this.props;
+      const divisor = sliderValues.reduce(
```

Comment:
This is to prevent the colors from getting too bright.  We take equal amounts from every channel.

Reply:
Do you think this is intuitive to the end user?

Reply:
I updated this.  In the shaders, I was adding together actual colors when in reality, we want to add together the intensities.